src/Domain/Contracts/FileProcessor.php

```diff
@@ -9,4 +9,8 @@
 interface FileProcessor
 {
     public function processFile(SplFileInfo $splFileInfo): void;
+
+    public function support(Insight $insight): bool;
```

Comment:
Just a quick little thing, could you add some docs to explain what this methods does.

I guess it's to check if a file processor supports the Insight?

Reply:
Yes you are right :wink: Added documentation